taqueria-plugin-tezos-client/client.ts

```diff
@@ -0,0 +1,263 @@
+import {Sandbox as theSandbox, execCmd, getArch, sendAsyncErr, sendErr, sendRes, sendAsyncRes, sendJsonRes, sendAsyncJsonRes} from '@taqueria/node-sdk'
+import type {SanitizedArgs, Config, Attributes, SandboxConfig, Failure, LikeAPromise, Sandbox, AccountDetails, StdIO, PluginResponse} from "@taqueria/node-sdk/types"
+import type {ExecException} from 'child_process'
+import retry from 'async-retry'
+import {join} from 'path'
+import glob from 'fast-glob'
+
+type Opts = SanitizedArgs & {sandboxName?: string}
+
+const isSandboxRunning = (sandboxName: string) =>
```

Comment:
@jchenche there looks to be quite a bit of copy/paste from the @taqueria/plugin-flextesa plugin. 

I think it would be best to add the @taqueria/plugin-flextesa as a dependency to the package.json file and then just import the functions you need into scope.

Reply:
After discussion with @mweichert. We are leaving them as is. Reason being that there's only 2 being copied over and they are tiny.